mix.exs

```diff
@@ -47,7 +43,16 @@ defmodule NervesPack.MixProject do
       {:vintage_net_direct, "~> 0.7.0"},
       {:vintage_net_ethernet, "~> 0.7.0"},
       {:vintage_net_wifi, "~> 0.7.0"},
-      {:vintage_net_wizard, "~> 0.2.0"}
+      # Optional Dependencies
+      # Include busybox if using nerves_system_* < 1.10.0
+      {:busybox, "~> 0.1", optional: true},
```

Comment:
Does `:busybox` even need to be listed? Since `nerves_pack` doesn't make any calls to Busybox even when it's present, it seems like it shouldn't care whether its available or not.

Reply:
agreed.